index.html

```diff
@@ -4,7 +4,76 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+
+    <link rel="stylesheet" href="css/style.css" />
+
     <title>Portfolio</title>
   </head>
-  <body></body>
+  <body>
+    <header>
+      <div class="logo">
+        <a href="#">
+          <h1>My Logo</h1>
+        </a>
+      </div>
+      <div>
+        <nav>
+          <span class="bars">
+            <i class="fa-solid fa-bars"></i>
+          </span>
+        </nav>
+      </div>
```

Comment:
[OPTIONAL] Great job implementing the changes required 👍 . I think the div tag wrapped around the nav tag is not necessary since it is no doing anything. Kindly remove it so as to make your code lighter . 😊

Reply:
done :heavy_check_mark: